alleycats-core/src/main/scala/alleycats/std/iterable.scala

```diff
@@ -1,13 +1,18 @@
 package alleycats
 package std
 
-import cats.{Eval, Foldable, Monoid}
+import cats.data.Chain
+import cats.kernel.instances.StaticMethods.wrapMutableIndexedSeq
+import cats.{Applicative, Eval, Foldable, Monoid, Traverse}
 
 object iterable extends IterableInstances
 
 trait IterableInstances {
-  implicit val alleycatsStdIterableFoldable: Foldable[Iterable] =
-    new Foldable[Iterable] {
+  @deprecated("use alleycatsStdIterableTraverse", "2.7.1")
+  val alleycatsStdIterableFoldable: Foldable[Iterable] = alleycatsStdIterableTraverse
+
+  implicit val alleycatsStdIterableTraverse: Traverse[Iterable] =
```

Comment:
Annoying, but this may need to be a `def` for bincompat.

```suggestion
  implicit def alleycatsStdIterableTraverse: Traverse[Iterable] =
```

Reply:
Oh I missed this - will that allow avoiding a BinCompat trait?

Reply:
Yup, I think so. TIL about the BinCompat trait trick though :)

Reply:
I think your way is cleaner 😀